alertlogic-cli

```diff
@@ -27,70 +30,96 @@ def import_dependencies():
         commands.append(alertlogiccli.commands.deployment.metadata)
     except ImportError:
         pass
-    
+
     try:
         import alertlogiccli.commands.troubleshooting
         commands.append(alertlogiccli.commands.troubleshooting.metadata)
     except ImportError:
         pass
-        
+
     return commands
 
+
 def make_parser(commands):
-    parser = argparse.ArgumentParser(description="alertlogic cloud insight client")
-    
-    parser.add_argument("-c", "--config_file", default=alertlogiccli.cons.DEFAULT_CONFIG_FILE,
-                                               help="use a specific configuration file")
-    parser.add_argument("-e", "--credentials_file", default=alertlogiccli.cons.DEFAULT_CREDENTIALS_FILE,
-                                                    help="use a specific credentials file")
-    parser.add_argument("--logging_config_file", help="use a specific configuration file for logging")
-    parser.add_argument("-p", "--profile", default=alertlogiccli.cons.DEFAULT_PROFILE,
-                                           help="use a specific profile in config and credentials")
-    parser.add_argument("--api_endpoint", help="alertlogic api endpoint, either: uk or us")
-    parser.add_argument("-a", "--account_id", help="use a specific account (managed accounts only)")
-    parser.add_argument("-d", "--deployment_id", help="deployment id used by most commands (uuid)")
-
-    subparsers = parser.add_subparsers(title="commands", description="valid commands")
-    
+    parser = argparse.ArgumentParser(
+        description="alertlogic cloud insight client")
+
+    parser.add_argument(
+        "-c",
+        "--config_file",
+        default=alertlogiccli.cons.DEFAULT_CONFIG_FILE,
+        help="use a specific configuration file")
+    parser.add_argument(
+        "-e",
+        "--credentials_file",
+        default=alertlogiccli.cons.DEFAULT_CREDENTIALS_FILE,
+        help="use a specific credentials file")
+    parser.add_argument(
+        "--logging_config_file",
+        help="use a specific configuration file for logging")
+    parser.add_argument(
+        "-p",
+        "--profile",
+        default=alertlogiccli.cons.DEFAULT_PROFILE,
+        help="use a specific profile in config and credentials")
+    parser.add_argument(
+        "--api_endpoint", help="alertlogic api endpoint, either: uk or us")
+    parser.add_argument(
+        "-a",
+        "--account_id",
+        help="use a specific account (managed accounts only)")
+    parser.add_argument(
+        "-d",
+        "--deployment_id",
+        help="deployment id used by most commands (uuid)")
+
+    subparsers = parser.add_subparsers(
+        title="commands", description="valid commands")
+
     for command in commands:
-        subparser = subparsers.add_parser(command["name"], help=command["help"])
+        subparser = subparsers.add_parser(
+            command["name"], help=command["help"])
```

Comment:
imo, breaking line this way when it is not really required is not adding to readability(i might be wrong in python world, so we can discuss it).